include/crm/common/results.h

```diff
@@ -107,6 +107,9 @@ enum pcmk_rc_e {
     /* When adding new values, use consecutively lower numbers, update the array
      * in lib/common/results.c, and test with crm_error.
      */
+    pcmk_rc_sim_failed          = -1031,
```

Comment:
Let's call this pcmk_rc_invalid_transition since that's how it will appear to users